lib/src/swipable_stack.dart

```diff
@@ -327,11 +363,14 @@ class _SwipableStackState extends State<SwipableStack>
             }
             widget.controller._updateSwipe(
               _SwipableStackPosition(
-                local: d.localPosition,
+                realLocal: d.localPosition,
                 start: d.globalPosition,
-                current: d.globalPosition,
+                real: d.globalPosition,
+                animationValue: 0,
               ),
             );
+
+            _dragStartController.forward(from: 0);
```

Comment:
Yes, should I change that?

Reply:
Oh well, now it comes back to me. I did it this way because there are fewer cases to consider this way and it is less error prone. Because I also calculate the current position always depending on the value of this controller. That means would have to set its value to 1 manually then, which would be a bit more efficient, but also more cumbersome.

Reply:
@MaddinMade 
Sorry again for the delay. I thought there was a more obvious approach, but I think this is fine for now.
If possible, I would appreciate it if you could leave a comment on how this code behaves.